virt/virtual_machines/advanced_vm_management/virt-understanding-aaq-operator.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY                               
+[id="virt-understanding-aaq-operator"]                                     
+= Configuring the Application-Aware Quota (AAQ) Operator                                               
+include::_attributes/common-attributes.adoc[]                   
+:context: virt-understanding-aaq-operator                                    
+                                                                
+toc::[]   
+
+You can use the Application-Aware Quota (AAQ) Operator to customize and manage resource quotas for individual components in an {product-title} cluster.
+
+include::modules/virt-about-aaq-operator.adoc[leveloffset=+1]
+
+include::modules/virt-enabling-aaq-operator.adoc[leveloffset=+1]
+
+include::modules/virt-configuring-aaq-operator.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
+
+* xref:../../../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[Resource quotas per project]
+* xref:../../../applications/quotas/quotas-setting-across-multiple-projects.adoc#quotas-setting-across-multiple-projects[Resource quotas across multiple projects]
+* xref:../../../rest_api/schedule_and_quota_apis/resourcequota-v1.adoc#resourcequota-v1[`ResourceQuota` API reference]
+* xref:../../../rest_api/schedule_and_quota_apis/clusterresourcequota-quota-openshift-io-v1.adoc#clusterresourcequota-quota-openshift-io-v1[`ClusterResourceQuota` API reference]
+* xref:../../../rest_api/workloads_apis/pod-v1.adoc#pod-v1[`Pod` object specification]
```

Comment:
It's "Pod object specification"

Reply:
So should I leave the link text as is?

Reply:
Change the link to https://docs.openshift.com/container-platform/4.17/rest_api/workloads_apis/pod-v1.html#spec-schedulinggates with the right text

Reply:
Thanks you! :)